platformio.ini

```diff
@@ -9,6 +9,27 @@ build_flags =
     -D PIO_FRAMEWORK_EVENT_QUEUE_PRESENT
     -D MBED_BUILD_PROFILE_RELEASE
     -D ROS_NOETIC_MSGS=1
+    -DKINEMATIC_TYPE=0 ;differential drive
+    
+platform_packages =
+    framework-mbed @ ~6.51506.0
+
+extra_scripts = 
+    post:gen_hex.py
+
+
+[env:core2_mec]
+platform = ststm32
+framework = mbed
+board = core2
+
+build_flags = 
+    -I$PROJECTSRC_DIR/TARGET_CORE2
```

Comment:
czy to jest potrzebne? Przy core2_diff tego nie ma

Reply:
```suggestion

```

Reply:
Wydaje mi się, że jest. core2_diff i  core2_mec powinny różnić się tylko jedną flagą KINEMATIC_TYPE.